test/Elastic.Apm.Tests/Mocks/TestLogger.cs

```diff
@@ -16,6 +16,9 @@ internal class TestLogger : ConsoleLogger
 
 		public TestLogger(LogLevel level, StringWriter writer) : base(level, writer, writer) => _writer = writer;
 
-		public List<string> Lines => _writer.GetStringBuilder().ToString().Split(Environment.NewLine, StringSplitOptions.RemoveEmptyEntries).ToList();
+		public List<string> Lines => _writer.GetStringBuilder()
+			.ToString()
+			.Split(Environment.NewLine.ToCharArray(), StringSplitOptions.RemoveEmptyEntries)
```

Comment:
`Environment.NewLine.ToCharArray()` was needed to build on `net461`, the parameter list used originally does not exist on `net461`.